src/behaviors/Column.js

```diff
@@ -224,17 +236,17 @@ Column.prototype = {
             return 'unknown';
         }
         var type = this.typeOf(value);
-        var isNumber = ((typeof value) === 'number');
+        //var isNumber = ((typeof value) === 'number');
```

Comment:
I was thinking we would include `getType` as sample code; and pull out `columnSchemaFactory` and provide that as sample code as well.